model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -377,8 +377,24 @@ def _extract_text_input_length(
     def _update_perf_config_based_on_parameter_combination(
         self, perf_config: PerfAnalyzerConfig, parameter_combination: Dict
     ) -> None:
+        if "request-parameter" in parameter_combination:
+            request_parameter = parameter_combination["request-parameter"]
+            max_token = self._extract_max_token_from_request_parameter(
```

Comment:
`max_token` -> `max_tokens` for consistency
```suggestion
            max_tokens = self._extract_max_tokens_from_request_parameter(
```